iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/LocationShifter.java

```diff
@@ -102,7 +113,9 @@ private LineData getOrCreateLinesData(Integer originalLine) {
   }
 
   static class LineData {
-    private Integer targetLine;
+    private Integer targetStartLine;
+    @Nullable
+    private Integer targetEndLine;
```

Comment:
Maybe it's better to store `targetEndLine = targetStartLine` instead of null Integer? Then line 83 can be simplified.